upgrade of all npm dependencies

capricube opened the pull request:
only had to change N3Util.isUrl() call to N3Util.isIRI()

capricube commented:
i'm having a strange "works on my machine" moment ... need to investigate a bit more

mcollina commented on .travis.yml:
```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-- 0.10
+- 5
```
This should be 0.10, 0.12, 4, 5

mcollina commented:
Let me know when it's done :).

capricube commented:
finally i found the problem. it was a hard nut to crack!

FYI: in order to make the tests work with the updated dependencies i had to downgrade level-test from 2.0.1 to 1.7.0. The tests didn't work on travis because they where executed using the in-memory leveldown (memdown) implementation. When i executed it locally i always had a leveldown dependency and everything worked. Only after i changed the `test/helper.js` file i was able to reproduce the travis errors locally. However i cannot say what the exact reason is why it doesn't work with memdown. I also added leveldown as devDependency so memdown is not being used.

mcollina commented on .travis.yml:
```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-- 0.10
+  - "5"
```
this should be 0.10, 0.12, 4, 5

mcollina commented on package.json:
```diff
@@ -32,24 +32,25 @@
+    "lodash": "2.4.1",
+    "mocha": "2.4.5",
+    "uglify-js": "2.6.2",
+    "zuul": "3.10.1"
```
can you please restore all the dependencies to be with `~`?

mcollina commented on test/iri_spec.js:
```diff
@@ -37,7 +37,7 @@ describe('IRI', function() {
       });
     });
-  });
+  }); 
```
unneeded space

mcollina commented:
Good work! There are just a couple of nits to fix.

capricube commented:
do we really need support for node 0.10 and 0.12? Those two are failing...

mcollina commented:
0.10 and 0.12 are currently supported by this library and by levelgraph (https://github.com/mcollina/levelgraph/blob/master/.travis.yml)

So, we should keep supporting them.

capricube commented:
if you merge my pull request for levelgraph i can upgrade the levelgraph depencency and the build will be fixed.

mcollina commented:
I've merged and released the levelgraph dep.

mcollina commented:
Done but tests are now failing everywhere apart from v5.

capricube commented:
will test again today in the evening
